Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbarSettings.h

```diff
@@ -5,6 +5,18 @@
 
 #include "SpatialGDKEditorToolbarSettings.generated.h"
 
+USTRUCT()
+struct FInteropClassList
+{
+	GENERATED_BODY()
+
+	UPROPERTY(EditAnywhere, config, Category = "Configuration", meta = (ConfigRestartRequired = false))
+	UClass* Actor;
```

Comment:
This could also be an `ActorComponent`. Maybe change to `ReplicatedClass` or `Class`?